alphagov/paas-cf
# [#109031802] Deploy concourse via bosh-init

# What
Story: https://www.pivotaltracker.com/story/show/109031802

Create a a job that runs inside the bootstrap vagrant Concourse that uses bosh-init to install the production Concourse.

# How to review

* Install the vagrant concourse (For example from the tutorial: https://github.com/actionjack/concourse-tutorial)
* Upload the pipeline:

```
BRANCH=109031802_deploy_concourse concourse/scripts/deploy-pipeline.sh <environment> create-deployer concourse/pipelines/create-deployer.yml <db password> <web gui password>
```

* From the GUI, unpause the job and start it
* It should deploy a Concourse at http://*environment*-concourse.cf.paas.alphagov.co.uk:8080/
* Username: admin, password as above

# Who can review
Anyone but @saliceti, @jimconner or @combor
